### DISCUSSION TITLE
New workflow -> Node.js seems to be missing pull_request trigger

### DISCUSSION THREAD

**Discussion by brody2consult** [community/community#25156] [Actions]
<p>If I click the “New workflow” button then click on “Set up this workflow” for “Node.js”, it seems to have trigger for push but not for pull_request.</p>
<p>But if I follow the lower-left link in “Node.js”, to <a href="https://github.com/actions/starter-workflows/blob/master/ci/node.js.yml" rel="nofollow noopener">here</a> then it does seem to have actions for both push and pull_request.</p>
<p>I hope we can get this fixed in the near future.</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Thanks! We have a script to sync the starter-workflows repo and what’s shown on <a href="http://github.com" rel="nofollow noopener">github.com</a>. Will be updated next time that runs (soon).</p>

**Comment by brody2consult** [comment]
<p>This issue seems to be resolved now. Thanks!</p>
